nbgitpuller/handlers.py

```diff
@@ -151,14 +174,15 @@ def get(self):
         repo = self.get_argument('repo')
         branch = self.get_argument('branch', None)
         depth = self.get_argument('depth', None)
+        compressed = self.get_argument('compressed', "false")
```

Comment:
Let's change this instead to a provider, and use it to call appropriate plugin? that way, compression becomes an implementation detail for the provider (see https://github.com/jupyterhub/nbgitpuller/pull/194/files#r685395876)

Reply:
yes removed this all good